src/test/java/lotto/LottoFactoryTest.java

```diff
@@ -0,0 +1,44 @@
+package lotto;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class LottoFactoryTest {
+    @DisplayName("발급된 로또 번호는 1~45 사이의 값 중 6개 이다.")
+    @Test
+    void publicTicket() {
+        Lotto lotto = LottoFactory.auto();
+        List<LottoNumber> lottoNumber = lotto.getLottoNumbers();
+        assertThat(lottoNumber.size()).isEqualTo(6);
```

Comment:
```suggestion
        assertThat(lottoNumber.size()).isNotZero();
        혹은
        assertThat(lottoNumber.size()).isEqualTo(LottoFactory.LOTTO_SIZE);
```

`private static final int LOTTO_SIZE = 6;` 
코드 처럼 `private`으로 설정 해주셨습니다.
그렇다는것은 외부에서 알지 않아야 한다고 규정해 놓은것이라 저는 생각했어요! 🤔 

그런데 외부에서 `.isEqualTo(6);` 이라는 조건을 검증하고 있어요.
이는 외부에서 과도하게 정보를 알려고 `예측` 하는것 아닐까요? 🤔 
만약, `6`에 대한 조건이 변경되는 상황에서 예측못하는 부분에서 에러가 발생할 수 있구요. 😢 

꼭 테스트 하려면 외부에서도 알수 있도록 `public`으로 오픈시키는게 좋을것 같기도 하네요. (개인적인 관점입니다)